app/helpers/foreman_tasks/tasks_helper.rb

```diff
@@ -2,16 +2,7 @@ module ForemanTasks
   module TasksHelper
     def format_task_input(task, include_action = false)
       return '-' unless task
-      parts = []
-      parts << task.get_humanized(:name) if include_action
-      parts << Array(task.get_humanized(:input)).map do |part|
-        if part.is_a? Array
-          part[1][:text]
-        else
-          part.to_s
-        end
-      end.join('; ')
-      parts.join(' ')
+      task.action || task.format_input(include_action)
```

Comment:
Helper shouldn't know how `action` field is generated. I would prefer leaving this calculation to the model:
``` ruby
# In task.rb:
class Task < ActiveRecord::Base
  def action
    super || format_input
  end
end
```